scripts/prep/subseasonal/exevs_subseasonal_obs_prep.sh

```diff
@@ -52,12 +52,12 @@ if [ $SENDMAIL = YES ] ; then
 fi
 
 # Cat the METplus log file
-log_dir=$DATA/$STEP/METplus_output/logs
-log_file_count=$(find $log_dir -type f |wc -l)
-if [[ $log_file_count -ne 0 ]]; then
-    for log_file in $log_dir/*; do
-	echo "Start: $log_file"
-	cat $log_file
-	echo "End: $log_file"
-    done
-fi
+#log_dir=$DATA/$STEP/METplus_output/logs
+#log_file_count=$(find $log_dir -type f |wc -l)
+#if [[ $log_file_count -ne 0 ]]; then
+    #for log_file in $log_dir/*; do
+	#echo "Start: $log_file"
+	#cat $log_file
+	#echo "End: $log_file"
+    #done
+#fi
```

Comment:
@PerryShafran-NOAA Do you want these sections commented out or removed entirely? I know NCO doesn't like having a lot of commented out code. CC @ShannonShields-NOAA @ShelleyMelchior-NOAA

Reply:
Removal is  better.  I know that I like to comment them out for testing (in case something goes wrong and we need to bring it back), but yeah, it's best to remove this entirely.  

@ShannonShields-NOAA Can we confirm that the entire log is being written out as METplus is being run?

Reply:
@PerryShafran-NOAA and @AliciaBentley-NOAA So first, I can remove the log sections and just keep a note for myself on how to put them back in for testing purposes. Here's the thing with METplus output...because I am running so many parallel jobs together, the output in the single output (.o*) file is all kind of mashed together like below:
06/17 17:17:43.216 metplus.7b92c8fa INFO: * Running METplus PB2NCWrapper
06/17 17:17:43.216 metplus.7b92c8fa INFO: *  at valid time: 2024-06-01 00:00
06/17 17:17:43.216 metplus.7b92c8fa INFO: ****************************************
06/17 17:17:43.216 metplus.7b92c8fa INFO: Processing forecast lead 0 hours
06/17 17:17:43.217 metplus.46049970 INFO: Running wrapper: GenEnsProd
06/17 17:17:43.217 metplus.f1c75af4 INFO: Running wrapper: GenEnsProd
So for testing purposes, it is highly preferred to keep the individual METplus log files.

Reply:
We did notice that for mpi runs, the log output seems to be all jumbled together. The issue is that with the LOG_TO_TERMINAL_ONLY set to True, it doesn't even create log files.  It would be a great diagnostic tool to separate them out if need be when there are errors, then we can set this value to False and then create the separate logs.  

One thing I noticed is that each run seems to have its own log ID (i.e. metplus.7b92c8fa).  Thus way we have some way of indentifying specific processes.  But not each line has that.  We may need to ask DTC to see if they can add this to every line of the log, if possible.  

In the meantime, for PR purposes, we'd like to remove all commented out lines like these.

@ShannonShields-NOAA @AliciaBentley-NOAA @ShelleyMelchior-NOAA

Reply:
Ok, will do.